wallet/keystores/encrypted_keystore.md

```diff
@@ -12,26 +12,29 @@ class EncryptedKeystore:
     static new_from_mnemonic(mnemonic: Mnemonic, password: string, randomness: Optional[Randomnness]): EncryptedKeystore;
 
     // Importing "constructor"
-    static import_from_object(object: KeyfileObject, password: string): EncryptedKeystore;
+    static import_from_object(object: KeyfileObject): EncryptedKeystore;
 
     // Importing "constructor"
-    static import_from_file(path: Path, password: string): EncryptedKeystore;
+    static import_from_file(path: Path): EncryptedKeystore;
 
     // When kind == 'secretKey', only index == 0 and passphrase == "" is supported.
     // When kind == 'mnemonic', secret key derivation happens under the hood.
     // Below, "passphrase" is the bip39 passphrase required to derive a secret key from a mnemonic (by default, it should be an empty string).
-    get_secret_key(index: int=0, passphrase: string=""): SecretKey;
+    decrypt_secret_key(index: int = 0, passphrase: string = ""): SecretKey;
+
+    // decyripts the secrect keys for a range of indexes; only for kind == 'mnemonic'
+    decrypt_secret_keys_in_range(start: int = 0, end: int = 10, passphrase: str = ""): SecretKey[];
```

Comment:
_Start_ and _end_ should not be optional.

Reply:
done